microservices/cart/src/commands/sagas/callbacks.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from collections import (
+    defaultdict,
+)
+
+from minos.common import (
+    Model,
+    ModelType,
+)
+from minos.saga import (
+    SagaContext,
+)
+
+_ReserveProductsQuery = ModelType.build("ValidateProductsQuery", {"quantities": dict[str, int]})
```

Comment:
I think `ReserveProductsRequest` is a better name in this case:
```suggestion
ReserveProductsRequest = ModelType.build("ReserveProductsRequest", {"quantities": dict[str, int]})
```